runtime/ftplugin/zig.vim

```diff
@@ -1,44 +1,44 @@
 " Vim filetype plugin file
 " Language: Zig
-" Upstream: https://github.com/ziglang/zig.vim
```

Comment:
So no more upstream?

Reply:
The changes here differ significantly enough from upstream that it is not really a copy of the upstream plugin at all anymore. Perhaps we can leave a comment that the ftplugin is _based_ on the upstream plugin though.

Reply:
Then please also add: `Maintainer: This runtime file is looking for a new maintainer.` (similar to other files):

Reply:
I am happy to act as maintainer (unless @Tiseno would like to). I upstreamed the original runtime files.

Reply:
yes exactly like this.